pkg/capacity/manager.go

```diff
@@ -21,7 +21,7 @@ type (
 
 func hasNotReachedMaxNumberOfSpacesThreshold(counts counter.Counts) spaceProvisionerConfigPredicate {
 	return func(spc *toolchainv1alpha1.SpaceProvisionerConfig) bool {
-		numberOfSpaces := uint(counts.SpacesPerClusterCounts[spc.Spec.ToolchainCluster])
+		numberOfSpaces := uint(counts.SpacesPerClusterCounts[spc.Spec.ToolchainCluster]) // nolint:gosec
```

Comment:
This fix is for the linter error . This error is thrown for integer to uint conversion
"pkg/capacity/manager.go:24:25: G115: **integer overflow conversion int -> uint (gosec)**
  		numberOfSpaces := uint(counts.SpacesPerClusterCounts[spc.Spec.ToolchainCluster])"
  		
  There are few other places where this error occurs, there is a fix for this as per [this issue ](https://github.com/securego/gosec/issues/1185)but it seems its not useful for us, There can be other option of using only Int or UInt and change it everywhere, i chose to use ignore this, if otherwise will try to see more for converting the whole thing

Reply:
ok, I see. Please add a comments to the lines explaining why it is OK to ignore the linter - why the number cannot overflow the capacity of the type

Reply:
updated it with comments